test_runner/regress/test_broken_timeline.py

```diff
@@ -116,6 +127,8 @@ def test_fix_broken_timelines_on_startup(neon_simple_env: NeonEnv):
     env.neon_cli.pageserver_stop(immediate=True)
     env.neon_cli.pageserver_start()
 
-    # Check that tenant with "broken" timeline is not loaded.
-    with pytest.raises(Exception, match=f"Failed to get repo for tenant {tenant_id}"):
-        env.neon_cli.list_timelines(tenant_id)
+    # Tenant is still not spoiled, so that is expected to be loaded and serve its timeline requests
```

Comment:
Hmm, so the new timeline will not be there after restart, but the tenant is still OK.

```suggestion
    # Creating the timeline didn't finish. The other timelines on tenant should still be present and work normally.
```

Reply:
Later, in cases we'll have non-tmp broken timeline, we could initialize it with similar, broken status.
It requires more work though.